test/helpers.js

```diff
@@ -16,8 +16,13 @@
       (mediaDevices.getUserMedia || function(){}).bind(mediaDevices);
   var ImageCapture = window.ImageCapture;
   var MediaRecorder = window.MediaRecorder;
-  var mediaRecorderStart = MediaRecorder.prototype.start;
-  var mediaRecorderStop = MediaRecorder.prototype.stop;
+  var mediaRecorderStart;
+  var mediaRecorderStop;
```

Comment:
Maybe we can just initialize these to something like, `mediaRecorderStart = window.MediaRecorder ? window.MediaRecorder.prototype.start : () => {}`?